docs/development.rst

```diff
@@ -244,37 +261,74 @@ This plugin produces a coverage report, example::
     mslib/msui/flighttrack.py                  383    117    141     16    66%
 
 
+Profiling
++++++++++
+
 Profiling can be done by e.g.::
 
-   $ python -m cProfile  -s time ./mslib/mswms/demodata.py > profile.txt
+   $ python -m cProfile  -s time ./mslib/mswms/demodata.py --seed > profile.txt
 
 example::
 
-    /!\ existing server config: "mss_wms_settings.py" for demodata not overwritten!
+   /!\ existing server config: "mss_wms_settings.py" for demodata not overwritten!
 
 
-    To use this setup you need the mss_wms_settings.py in your python path e.g.
-    export PYTHONPATH=$HOME/mss
-             398119 function calls (389340 primitive calls) in 0.834 seconds
+   /!\ existing server auth config: "mss_wms_auth.py" for demodata not overwritten!
 
-       Ordered by: internal time
 
-       ncalls  tottime  percall  cumtime  percall filename:lineno(function)
-           19    0.124    0.007    0.496    0.026 demodata.py:912(generate_file)
-           19    0.099    0.005    0.099    0.005 {method 'close' of 'netCDF4._netCDF4.Dataset' objects}
+   To use this setup you need the mss_wms_settings.py in your python path e.g.
+   export PYTHONPATH=~/mss
```

Comment:
Is a command missing?

Reply:
Because this is the output from profiling ./mslib/mswms/demodata.py --seed 

We could improve by rich the formatting, but that's all